P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -2269,6 +2085,265 @@ occupation and residence.</p>
+              <name>Eastern Front</name>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Eastern_Front_(World_War_II)"/>
+            </place>
+            <place xml:id="l-Zam">
+              <country>Zambia</country>
+            </place>        
+          </listPlace>
+          <!-- ========= people ========= -->
+          <listPerson>
+            <person xml:id="p-VV">
+              <persName xml:lang="sr-Cyrl">Весна Вуловић</persName>
+              <persName xml:lang="sr-Latn">Vesna Vulović</persName>
+              <birth when="1950-01-03"/>
+              <death when="2016-12-23"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Vesna_Vulovi%C4%87"/>
+            </person>
+            <person xml:id="p-IC">
+              <persName xml:lang="uk-Cyrl">Іван Михайлович Чиссов</persName>
+              <persName xml:lang="ru-Cyrl">Иван Михайлович Чисов</persName>
+              <persName xml:lang="uk-Latn">Ivan Mikhailovich Chisov</persName>
+              <birth when="1916"/>
+              <death when="1986"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Ivan_Chisov"/>
+            </person>
+            <person xml:id="p-AM">
+              <persName>Alan Eugene Magee</persName>
+              <birth when="1919-01-13"/>
+              <death when="2003-12-20"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Alan_Magee"/>
+            </person>
+            <person xml:id="p-NA">
+              <persName>Nicholas Stephen Alkemade</persName>
+              <birth when="1922-12-10"/>
+              <death when="1987-06-22"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Nicholas_Alkemade"/>
+            </person>
+            <person xml:id="p-BG">
+              <persName>Edward Michael Grylls</persName>
+              <birth when="1974-06-07"/>
+              <ptr type="furtherInfo" target="https://www.beargrylls.com/pages/about-bear-grylls"/>
+            </person>
+            <person xml:id="p-JK">
+              <persName>Juliane Koepcke</persName>
+              <birth when="1954-10-10"/>
+              <ptr type="furtherInfo" target="https://en.wikipedia.org/wiki/Juliane_Koepcke"/>
+            </person>
+            <person xml:id="p-JB">
+              <persName>James Boole</persName>
+              <birth>1978, give or take a year</birth>
+              <ptr type="furtherInfo" target="https://www.facebook.com/james.boole/?ref=page_internal"/>
+            </person>
+          </listPerson>
+          <!-- ========= conjunctions and disjunctions ========= -->
+          <altGrp type="location">
+            <alt xml:id="l-KorZ" target="#l-Ken #l-Zam"/>
+          </altGrp>
+        </standOff>
+      </egXML>
+      <p>The <att>where</att> attribute on <gi>event</gi> can point to multiple 
+        canonical place descriptions, which should indicate 
```

Comment:
I find the "should" a bit confusing. I am perfectly fine with just "which indicate..."